### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### `TO DO LIST`
**In this milestone.**
I added the following:

>   Webpack (installed and added all the necessary files that the project requires)

> Html (Added the structure of the mockup)

> Css (Added styling to the mock-up structure)

> Linters (Added stylelint, eslint and a webhint files and codes )

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,13 +2,45 @@ import _ from 'lodash';
 import './style.css';
 
 function component() {
-    const element = document.createElement('body');
+  const element = document.createElement('div');
 
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['hello', 'webpack'], ' ');
-    element.classList.add('hello');
+  // Lodash, Now has imported new script
```
Comment (wandji20) [inline src/index.js:7]: - [ ] It will be nice to remove all commented code from your codebase.

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @gabrielcoder247  :raised_hands: 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this project.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Very descriptive commit messages and pull request summary
:heavy_check_mark: Excellently crafted README 
:heavy_check_mark:  Dynamic render of todos :100: 

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved
- [ ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .hintrc, .eslintrc.json etc).

**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (gabrielcoder247) [reply src/index.js:7]: Dear @wandji20, that code is not commented is a statement that is commented.
But anyway if you want me to remove the statement then fine or tell me exactly what you mean

**Review by EricMbouwe — APPROVED** [review]
### Hi @gabrielcoder247, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**